cookbooks/new_ping.md

```diff
@@ -53,65 +63,115 @@ field. The `schema` element has top-level fields (e.g. `Timestamp`, `Type`), as
 under the `Fields` element. Any of these can be used in the `metadata` section of your parquet schema,
 except for `submission`.
 
-Some common ones might be:
+Some common ones for Telemetry data might be:
 - `Date`
 - `submissionDate`
 - `geoCountry`
 - `geoCity`
+- `geoSubdivision1`
+- `geoSubdivision2`
 - `normalizedChannel`
 - `appVersion`
 - `appBuildId`
 
+And for non-Telemetry data:
+- `geoCountry`
+- `geoCity`
+- `geoSubdivision1`
+- `geoSubdivision2`
+- `documentId`
+
 *Important Note*: Schema evolution of nested structs is currently broken, so you will not be able to add
 any fields in the future to your `metadata` section. We recommend adding any that may seem useful.
 
 ### Testing The Schema
 
 Note that this only works if data is _already_ being sent, and you want to test the schema you're
-writing on the data that is currently being ingested.
+writing on the data that is currently being ingested. If your data isn't being sent yet, skip to the
+[Start Sending Data](#start-sending-data) section.
```

Comment:
We could link to the newly-available [edge validator](https://github.com/mozilla-services/edge-validator) for locally testing new schemas / pings.